.github/workflows/pullrequest-trusted.yml

```diff
@@ -5,7 +5,6 @@ concurrency:
   cancel-in-progress: true
 
 on:
-  workflow_dispatch:
```

Comment:
we aren't using this

Reply:
[opt] I believe I put this in there originally because "why not?" Made for easier testing if/when needed. Fine with removal though.